desktop/components/artwork_item/templates/artwork.jade

```diff
@@ -4,7 +4,7 @@
 - displayPurchase = displayPurchase || false
 - showBlurbs = showBlurbs || false
 - isAuction = isAuction || (set && set.get('type') === 'auction artworks') || false
-- hasInstitutionPartner = artwork.get('partner') && artwork.get('partner').type === 'Institution'
+- hasInstitutionPartner = artwork.get('partner') && artwork.get('partner').type === 'Institution' && !artwork.get('forsale')
```

Comment:
The `type` of this partner is `Institutional Seller` tho, (based on https://api.artsy.net/api/v1/artwork/pacita-abad-feel-the-beat).

So that shouldn't be subject to the whole 'hiding works from institutions override' here IMO.

It seems as though this is the issue that is causing it to not show up: https://github.com/artsy/force/commit/0b2c0d11cce9b3f4917ef7cd7b5c8c8d0d760837

It was a part of this innocuous PR: https://github.com/artsy/force-private/pull/4629/files

In an ironic twist indeed, it was authored by you! 

It looks as though it's explicitly hiding the availability message from `Institutional Sellers`. That seems like we can remove it (it is a seller after all), but I'm not aware of any history there. @starsirius does it ring a bell? Essentially we're treating `Institutional Seller` the same as `Institution` in terms of hiding prices, whereas it seems like (bc they are explicitly sellers) we can just treat them like any other partner? (and avoid the institution override).